specification/hugr.md

```diff
@@ -1168,44 +1168,78 @@ The new hugr is then derived as follows:
 
 ###### `Replace`
 
-This takes as input:
+This is the general subgraph-replacement method.
 
-  - a set S of IDs of nodes in Γ, all of which are separated;
+A _partial hugr_ is is a graph $G$ having the same node and edge types as a hugr
```

Comment:
Ah hang on (writing this after some of the comments below) - this is the second definition of 'partial hugr'. There's another one just *before* the API Methods section. That one's slightly more precise, but is it correct?

Reply:
Sorry, I forgot to delete that first one. Agree this second needs making more precise.